web/docs/operators/azure-log-analytics.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_custom_props:
+  operator:
+    source: false
+    sink: true
+---
+
+# azure-log-analytics
+
+Sends events via the [Microsoft Azure Logs Ingestion
+API](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/logs-ingestion-api-overview).
+
+## Synopsis
+
+```
+azure-log-analytics --tenant-id <tenant-id> --client-id <client-id>
+--client-secret <client-secret> --dce <data-collection-endpoint> --dcr
+<data-collection-rule-id> --table <table-name> [--batch-size <batch-size>]
+```
+
+## Description
+
+The `azure-log-analytics` operator makes it possible to upload events to
+[supported
+tables](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/logs-ingestion-api-overview#supported-tables)
+or to [custom tables](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/create-custom-table?tabs=azure-portal-1%2Cazure-portal-2%2Cazure-portal-3#create-a-custom-table)
+in Microsoft Azure.
+
+The operator handles access token retrievals by itself and updates that token
+automatically, if needed.
+
+### `<tenant-id>`
+
+The Microsoft Directory (tenant) ID, written as
+`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
+
+### `<client-id>`
+
+The Microsoft Application (client) ID, written as
+`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
+
+### `<client-secret>`
+
+The client secret.
+
+### `<data-collection-endpoint>`
+
+The data collection endpoint URL.
+
+### `<data-collection-rule-id>`
+
+The data collection rule ID, written as `dcr-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`.
+
+### `<table-name>`
+
+The table to upload events to.
+
+### `<batch-size>`
+
+The batch size for each upload request. Defaults to 8192.
```

Comment:
This should mention the 500MB limit per request, and also that this limit is expressed in number of events instead of number of bytes.

Reply:
Added in https://github.com/tenzir/tenzir/pull/4281/commits/0978fc38aa2668e38bb41980a7ac94c8ba7b491d

(It's actually 500 MB _per minute_)

Reply:
Sounds like we should have a "throttle" or "rate-limit" operator that can throttle pipelines until they're below some rate?

Reply:
Sounds good, but maybe the batch limit suffices for now.

Should I create an issue for the `throttle` operator?

Reply:
We already have one for a `rate-limit` operator.